net/usrsock/usrsock_sendmsg.c

```diff
@@ -142,9 +143,18 @@ static int do_sendto_request(FAR struct usrsock_conn_s *conn,
   {
   };
 
-  struct iovec bufs[2 + msg->msg_iovlen];
+  struct iovec *bufs;
+  size_t iovlen;
+  int ret;
   int i;
 
+  iovlen = 2 + msg->msg_iovlen;
+  bufs = kmm_malloc(sizeof(*bufs) * iovlen);
```

Comment:
could we use alloca or save the buffer to conn? it isn't good to do the allocation in the hot path.

Reply:
i guess we can have a special case for small iovlen.